build/common.ps1

```diff
@@ -313,7 +313,8 @@ Function New-Package {
 		[string]$ReleaseLabel = "zlocal",
 		[string]$BuildNumber,
 		[switch]$NoPackageAnalysis,
-		[string]$Version
+		[string]$Version,
```

Comment:
We used to have a NuGet package that pulled these in, however, it was filled with MSBuild magic which made the whole build process rather hard to understand/modify.
A git submodule may be a bit easier indeed, as it also facilitates traversing history. I could live with that.

Alternatively, we could just host the PowerShell scripts on a GitHub repo, and restore it on disk dynamically before loading it. Similar to how [`dnvm`](https://github.com/aspnet/dnvm) got restored from GitHub. That also provides us with the VCS benefits, and is a simple blob download.